compiler.rst

```diff
@@ -396,20 +414,22 @@ the magic number if you change the bytecode, while you are debugging your work
 you will be changing the bytecode output without constantly bumping up the
 magic number.  This means you end up with stale .pyc files that will not be
 recreated.
-Running ``find . -name '*.py[co]' -exec rm -f {} ';'`` should delete all .pyc
+Running ``find . -name '*.py[co]' -exec rm -f '{}' +`` should delete all .pyc
```

Comment:
Things I learned today: Usage of `+` in `find exec`. Is this widely supported? It does works on a Mac. I can check Cygwin if I can ever get out of Cleveland.

Reply:
It is the part of POSIX and I'm sure it is widely supported on systems used for developing future CPython versions.

It was not supported on some ancient commercial Unixes like SunOS 5.8 15 years ago: http://www.manpages.info/sunos/find.1.html.